java/src/main/java/com/saucelabs/saucebindings/configs/SauceConfigsChrome.java

```diff
@@ -0,0 +1,34 @@
+package com.saucelabs.saucebindings.configs;
+
+import com.saucelabs.saucebindings.SauceOptions;
+import org.openqa.selenium.chrome.ChromeOptions;
+
+public class SauceConfigsChrome extends SauceConfigsBrowser<SauceConfigsChrome> {
+    /**
+     * @deprecated Not for public use
+     *      Use: {@link SauceOptions#chrome()}} or {@link SauceOptions#chrome(ChromeOptions)}}
+     */
+    @Deprecated
+    public SauceConfigsChrome(ChromeOptions chromeOptions) {
+        sauceOptions = new SauceOptions(chromeOptions);
+    }
+
+    public SauceConfigsChrome setCapturePerformance(Boolean bool) {
```

Comment:
I did document a bug related to this https://github.com/saucelabs/sauce_bindings/issues/241

Reply:
latest commit throws an exception if `setCapturePerformance()` is called before `setName()` we can remove this if/when we implement the automatic naming idea for the test runners.